### ISSUE TITLE
Key error with submit to archive.org disabled

### ISSUE THREAD

**Issue by lhagan** [pirate/bookmark-archiver#53]
In `config.py`, I've set:

```
SUBMIT_ARCHIVE_DOT_ORG = os.getenv('SUBMIT_ARCHIVE_DOT_ORG', 'False').lower() == 'true'
```

Running `./archive.py ~/Downloads/bookmarks_export.html` yields the following error: 

```
File "/home/lhagan/bookmark-archiver/index.py", line 141, in write_html_link_index
    'archive_org': link['latest']['archive_org'] or 'https://web.archive.org/save/{}'.format(link['url']),
KeyError: 'archive_org'
```

Am I doing this wrong or have I encountered a bug here?

**Comment by pirate** [comment]
Nope, you found a bug, sorry about that! I just fixed it (08591e8), pull master and try again.

**Comment by pirate** [comment]
Closing this for now, comment back here if you're still having any problems.
